docs: apply tone guide to Overview.md (#1766)
carbon-design-system/carbon-ai-chat: packages/ai-chat/docs/Overview.md

## Overview

Carbon AI Chat is a backend-agnostic chat UI designed for modern AI-based flows. This page shows you how to install it, configure it, and connect it to your server.

> Try different settings in the [demo](../../demo/index.html), which shows the full-featured chat interface with its many customization options and response types.

Reviewer: ```suggestion
Try our [demo](../../demo/index.html), which shows the full-featured chat interface with its many customization options and response types.
```

I feel like just directing users to the demo here is good enough and the "different" settings can be brought up on our pages and is still referenced on the end of the sentence

## Using the API

### Picking your component

This package provides both React and web component versions of Carbon AI Chat. Each framework exports two versions. One uses a float layout — the classic launcher with the chat in the lower-right corner for customer service use cases. The other renders the chat into a container element you choose (sidebar, fullscreen, and so on), growing responsively to fill it.

> See the [React](./React.md), [web component](./WebComponent.md), and [Angular](./Angular.md) docs.

Reviewer: Little confusing reading this package provides both react and web components and then randomly has an "and Angular" at the end, we should clarify if we support angular or is this a way to use the chat in angular?

## Configuration

> Each component takes {@link PublicConfig} options as props that control how the chat looks and behaves before your users see it. Set where the chat renders along with its size, frame, and corners ({@link LayoutConfig}); choose which surfaces appear — the {@link LauncherConfig launcher} (IBM's or your own), {@link HeaderConfig header}, {@link HomeScreenConfig home screen}, and {@link InputConfig input}; control how the chat reaches your server ({@link PublicConfigMessaging}); and set its theme, language, and assistant identity. See {@link PublicConfig} for every option.

Reviewer: in the context of developers accept is more appropriate 
```suggestion
Each component accepts {@link PublicConfig} options as props that control how the chat looks and behaves before your users see it. Set where the chat renders along with its size, frame, and corners ({@link LayoutConfig}); choose which surfaces appear — the {@link LauncherConfig launcher} (IBM's or your own), {@link HeaderConfig header}, {@link HomeScreenConfig home screen}, and {@link InputConfig input}; control how the chat reaches your server ({@link PublicConfigMessaging}); and set its theme, language, and assistant identity. See {@link PublicConfig} for every option.
```

## Connecting to your server

The Carbon AI Chat can send messages to your server and retrieve conversations from it. It supports both streamed and non-streamed responses.

For details, see the [connect to your server](./CustomServer.md) guide.

## Customizing the view

> You can customize Carbon AI Chat at three levels, from quickest to deepest: configure behavior and appearance through {@link PublicConfig} props, restyle with a Carbon [theme](./Theming.md) and CSS custom-property tokens for color, sizing, and placement, or inject your own content into [slots](./WriteableElements.md), [custom panels](./CustomPanels.md), [responses](./Responses.md), and [message footers](./CustomMessageFooter.md).

Reviewer: We name three levels, prob best to use a list here?
Also Quickest -> Deepest sounds a bit confusing since it sounds like two different dimensions

## Cookies and GDPR

> The Carbon AI Chat does not use any cookies. It uses the browser's transient session storage for required behavior to track the Carbon AI Chat's state as you move from page to page — for example, whether the home screen should be visible, and so on.

Reviewer: I would change the word `transient` i didn't even know what it meant, maybe temporary?
